### ISSUE TITLE
When input parameter is just a zod string, it does not show up in the panel

### ISSUE THREAD

**Issue by datagutt** [iway1/trpc-panel#18]
When the procedure input is just a z.string(), there are no input parameters shown and no execute-button shown:
```
export const pageRouter = t.router({
  public: t.router({
    byId: procedure.input(z.string()).query(async ({ input: id }) => {
      return repository.findOne(id);
    }),

    getTree: procedure.input(z.string()).query(async ({ input: id }) => {
      return repository.getTree(id);
    }),
  }),
```

**Comment by iway1** [comment]
Thanks for reporting this I'll check it out 🙂

**Comment by iway1** [comment]
fixed in 1.2.1, can now support non `z.object()` top level input schemas.
